src/store/functions/fetchPostsAndUsers.js

```diff
@@ -0,0 +1,32 @@
+import { handleError, startLoading, handleSuccess } from '../actions';
+
+const loadPostsAndUsers = () => (dispatch) => {
+  const usersApi = 'https://jsonplaceholder.typicode.com/users';
```

Comment:
create one base url

Reply:
I don't understand what you mean, help, please

Reply:
create a variable that stores `https://jsonplaceholder.typicode.com` as a base url and reuse it everywhere you need  it